src/rest-client-v5.ts

```diff
@@ -1361,12 +1374,16 @@ export class RestClientV5 extends BaseRestClient {
    *
    * TIP:
    * The API key must have one of the permissions to be allowed to call the following API endpoint.
-   * - sub API key: "Account Transfer"
+   * - sub API key: "Account Transfer", "Sub Member Transfer"
+   * - master API Key: "Account Transfer", "Sub Member Transfer", "Withdrawal"
    *
-   * DANGER: BE CAREFUL! The API key used to call this interface will be invalid immediately.
+   * DANGER: BE CAREFUL! The sub API key used to call this interface will be invalid immediately.
    */
-  deleteSubApiKey(): Promise<APIResponseV3WithTime<{}>> {
-    return this.postPrivate('/v5/user/delete-sub-api');
+  deleteSubApiKey(apikey?: string): Promise<APIResponseV3WithTime<{}>> {
+    return this.postPrivate(
+      '/v5/user/delete-sub-api',
+      apikey ? { apikey } : undefined,
+    );
```

Comment:
```suggestion
  deleteSubApiKey(params?: { apikey?: string; ): Promise<APIResponseV3WithTime<{}>> {
    return this.postPrivate(
      '/v5/user/delete-sub-api',
      params,
    );
```

Even if there's only one param right now, I'd prefer to wrap it in an object. I used to do the same as you in the past, but then it becomes problematic when the endpoint is updated and more and more params are added (some may be optional while others are suddenly required). Using an object gives more future-proofed freedom.